iib/web/api_v1.py

```diff
@@ -16,6 +17,57 @@ def get_request():
     return 'Test request success!'
 
 
+@api_v1.route('/builds/<int:request_id>')
+def get_build(request_id):
+    """
+    Retrieve the build request.
+
+    :param int request_id: the request ID that was passed in through the URL.
+    :rtype: flask.Response
+    :raise NotFound: if the request is not found
+    """
+    query = Request.query.options(*Request.get_query_options(verbose=True))
+    return flask.jsonify(query.get_or_404(request_id).to_json())
+
+
+@api_v1.route('/builds')
+def get_builds():
+    """
+    Retrieve the paginated build requests.
+
+    :rtype: flask.Response
+    """
+    state = flask.request.args.get('state')
+    verbose = str_to_bool(flask.request.args.get('verbose'))
+    max_per_page = flask.current_app.config['IIB_MAX_PER_PAGE']
+
+    query = Request.query.options(*Request.get_query_options(verbose=verbose))
+    if state:
```

Comment:
I was wondering if we can move this validation to a static method in `RequestState` so that I can re-use this bit in the patch API too. thoughts?